### PR TITLE
Unit tests for filterByRegionMunicipalities

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard22**

### CODE REVIEW

File: packages/app/src/static-props/utils/__tests__/get-data.spec.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import * as assert from 'uvu/assert';
+
+const FilterByRegionMunicipalities = suite('filterByRegionMunicipalities');
+
+FilterByRegionMunicipalities('should filter municipalities by region', () => {
+  sinon
+    .stub(File, 'getVrGmCodesForGmCode')
+    .returns(['AMS', 'HRLM', 'ALKM', 'VLDM']);
+  const data = [{ gmcode: 'AMS' }, { gmcode: 'GR' }, { gmcode: 'VLDM' }];
+  const context = { params: { code: 'AMS' } };
```
Comment (VWSCoronaDashboard18) [inline packages/app/src/static-props/utils/__tests__/get-data.spec.ts]: Maybe use an other example as AMS has the same VR as GM as code.

File: packages/app/src/static-props/utils/__tests__/filter-by-region-municipalities.spec.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      (err: Error) => err.message === 'No municipalCode in context params'
+    );
+  }
+);
+
+FilterByRegionMunicipalities(
+  'should throw an error if no regionCodes are found for municipalCode',
+  () => {
+    sinon.stub(File, 'getVrGmCodesForGmCode').returns(undefined);
+    const data = [{ gmcode: 'AMS' }, { gmcode: 'GR' }, { gmcode: 'VLDM' }];
```
Comment (VWSCoronaDashboard24) [inline packages/app/src/static-props/utils/__tests__/filter-by-region-municipalities.spec.ts:39]: Not sure if it could be a use case, but maybe add a test case with empty `data` as well? 
```
const data = []
```

**Review by VWSCoronaDashboard24 — APPROVED** [review]
LGTM!

File: packages/app/src/static-props/utils/__tests__/filter-by-region-municipalities.spec.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+    assert.throws(
+      () => filterByRegionMunicipalities(data, context),
+      (err: Error) => err.message === `No regionCodes found for NH`
+    );
+  }
+);
+
+FilterByRegionMunicipalities(
+  'should return an empty array if data is empty',
```
Comment (VWSCoronaDashboard24) [inline packages/app/src/static-props/utils/__tests__/filter-by-region-municipalities.spec.ts:50]: 👍

**Review by VWSCoronaDashboard24 — APPROVED** [review]

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Looks great
